### ISSUE TITLE
Yarn build fails

### ISSUE THREAD

**Issue by superwhykz** [Shopify/polaris#442]
# Issue summary

I am trying to migrate to a new release v2.11.0  in dev env and getting the following error.  Just to mention prior to that I upgraded  https://reactjs.org/blog/2018/10/01/create-react-app-v2.html. 

```
15:42 $ make dev
yarn build:development; \
	firebase deploy -P development;
yarn run v1.10.1
$ REACT_APP_ENV=development yarn run build
$ sh -ac '. .env.${REACT_APP_ENV}; react-scripts build'
Creating an optimized production build...
Failed to compile.

./node_modules/@shopify/polaris/styles.css
Unclosed bracket (4173:4)

  4171 |   text-transform:none;
  4172 |   text-transform:initial;
> 4173 |   letter-spacing:normal;
       |    ^
  4174 |   letter-spacing:initial;
  4175 |   display:inline-flex;
  4176 |   align-items:center;


error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

## Expected behavior
Build

## Actual behavior
Fails to build

## Steps to reproduce the problem
1. `yarn build`

## Specifications

- Are you using the React components? (Y/N): Y
- Polaris version number: 2.11.0
- Browser: Version 69.0.3497.100 (Official Build) (64-bit)
- Device: Mac Pro 
- Operating System: macOS Siera

**Comment by mbaumbach** [comment]
I'm not seeing issues with CRA 2.0.3, Polaris 2.11.0, and Yarn 1.10.1. Looking at the styles.css file in my `node_modules`, I actually don't see that line at all. Perhaps deleting your `node_modules` folder and re-running `yarn` will correct the issue?

**Comment by superwhykz** [comment]
I did that and cleared the cache.  Let me try again.

**Comment by superwhykz** [comment]
@mbaumbach, that is pretty weird ..... I did  the same thing an hour ago and it did not work but now seems passing .... 

`Old attempt:`
```
✔ ~/GitHub/ui [feature/email-counter|✚ 33]
15:57 $ rm -rf node_modules/
✔ ~/GitHub/ui [feature/email-counter|✚ 33]
15:57 $ yarn cache clean
yarn cache v1.10.1
success Cleared cache.
✨  Done in 5.41s.
✔ ~/GitHub/ui [feature/email-counter|✚ 33]
15:57 $
✔ ~/GitHub/ui [feature/email-counter|✚ 33]
15:57 $ yarn install
yarn install v1.10.1
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
[4/4] 📃  Building fresh packages...
✨  Done in 128.87s.
✔ ~/GitHub/ui [feature/email-counter|✚ 33]
15:59 $
✔ ~/GitHub/ui [feature/email-counter|✚ 33]
16:01 $ make dev
yarn build:development; \
	firebase deploy -P development;
yarn run v1.10.1
$ REACT_APP_ENV=development yarn run build
$ sh -ac '. .env.${REACT_APP_ENV}; react-scripts build'
Creating an optimized production build...
Failed to compile.

./node_modules/@shopify/polaris/styles.css
Unclosed bracket (4173:4)

  4171 |   text-transform:none;
  4172 |   text-transform:initial;
> 4173 |   letter-spacing:normal;
       |    ^
  4174 |   letter-spacing:initial;
  4175 |   display:inline-flex;
  4176 |   align-items:center;


error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by mbaumbach** [comment]
Weird, glad it's working though!

**Comment by superwhykz** [comment]
Thanks so much!

**Comment by mbaumbach** [comment]
@superwhykz I left a comment on that ticket, you should be safe. I'm just a consumer/customer of Polaris though (read: Not a Shopify employee), so you might not rely on me providing "official" help here. Just trying to help out a fellow dev in this case. :)

**Comment by superwhykz** [comment]
ehh .... :) thanks much!!!!
